src/App.jsx

```diff
@@ -14,57 +15,64 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setGood] = useState("Jam");
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood
+          ? `${selectedGood} is selected`
+          : 'No goods selected'}
```

Comment:
There is a minor syntax issue here. The JSX expression `{selectedGood}` should be enclosed in parentheses to ensure proper rendering. Change it to `{selectedGood ? `${selectedGood} is selected` : 'No goods selected'}`.